### DISCUSSION TITLE
(web admin) - En el importador de properties, se puede dejar opcional la columna subproperty?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#191] [2. Questions and Answers] [answered]
Hola

Resulta que al usar el importador de properties del web admin, esté se cargará si y solo si, la base cuenta con subproperty. SI son nuevas propiedades no se pueden cargar dado que piden ese campo.

No sé si es un bug o una feature, por eso lo colocó aquí

Gracias :>

**Comment by acedward** [comment, marked as answer]
@NicolasPLG el importador está deprecado, y será eliminado de la interfaz durante los próximos días. Por lo tanto no está recibiendo bug fixes, ni mejoras :(

Existe un nuevo proyecto de importación, aun no es liberado públicamente. 
Pero puedes preguntar internamente para probarlo en beta.

**Reply by NicolasPLG** [reply]
@acedward  corresponde a lo que se presentó la vez pasada?. Cerraré esta discusión

Gracias @acedward

**Reply by MiikeVi** [reply]
> @acedward corresponde a lo que se presentó la vez pasada?. Cerraré esta discusión
> 
> Gracias @acedward

@NicolasPLG exacto, corresponde al importer de properties/users que se presentó hace un tiempo :) luego podemos hablar de eso para probarlo
